src/app/main.scss

```diff
@@ -284,6 +284,16 @@ select {
   }
 }
 
+code {
+  position: relative;
+  background: rgba(255, 255, 255, 0.05);
+  border: 1px solid rgba(255, 255, 255, 0.1);
+  border-radius: 3px;
+  padding: 0 4px;
+  transform: translateY(-4px);
```

Comment:
checking and unchecking this box in chrome style inspector does not seem to have any effect. what's intended here?

Reply:
This is the "code" style text for `deepsight:harmonizable` inside the tooltip 

<img width="203" alt="image" src="https://github.com/DestinyItemManager/DIM/assets/1396158/9572295c-c69b-4ff7-8e1d-52f36839d446">

Reply:
sorry github's interface makes it unclear this is in reference to line 293

Reply:
thanks - good catch, this is redundant